recipe/meta.yaml

```diff
@@ -0,0 +1,117 @@
+{% set name = "fairseq2" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # fairseq2 needs to be cloned from gh repo to pull .gitmodules and clone some of the necessary vendored dependencies
+  git_url: https://github.com/facebookresearch/fairseq2
+  git_rev: v0.2.0
+  patches:
+    - 0001-use-host-env-python.patch
+    - 0002-fix-unsupported-linker-option.patch
+    - 0003-unvendor-fmt.patch
+    - 0005-unvendor-pybind11.patch
+    - 0006-unvendor-gtest.patch
+    - 0007-unvendor-kaldi-native-fbank.patch
+    - 0008-find-python-in-host-environment.patch
+    - 0009-patch-out-failing-TestSampleOp-tests.patch  # [arm64 or aarch64]
+
+build:
+  # skip py312 because it's supported only from 0.2.1
+  skip: True  # [py<38]
+  skip: True  # [py>311]
+  # skip s390x because of missing libsndfile
+  skip: True  # [s390x]
+  # win officially not supported by upstream as per
+  # https://github.com/facebookresearch/fairseq2/blob/v0.2.0/README.md#installing-on-windows
+  skip: True  # [win]
+  missing_dso_whitelist:
+    - $RPATH/libtorch_cpu.so     # [linux]
+    - $RPATH/libtorch_python.so  # [linux]
+    - $RPATH/libc10.so           # [linux]
+    - $RPATH/libtorch_cpu.dylib     # [osx]
+    - $RPATH/libtorch_python.dylib  # [osx]
+    - $RPATH/libc10.dylib           # [osx]
+  ignore_run_exports:
+    - libsndfile
```

Comment:
This should be removed since you do have `libsndfile` in the runtime dependencies.

Reply:
will remove.